pyproject.toml

```
@@ -22,7 +23,7 @@ include = ["pyproject.toml"]
 pycln = "pycln.cli:app"
 
 [tool.poetry.dependencies]
-python = ">=3.6.2, <3.10"
+python = ">=3.6.2, <3.11"
```

```suggestion
python = ">=3.6.2"
```

**Please** never cap this. Ever. It causes failures and makes it impossible to test develop version of Python (or released version of Python or CI services like pre-commit.ci now). Python can never change its own version, so this will _never_ turn a failure into a success by changing the solve, which is the point of limits.

Also, poetry is really annoying here and will force everyone depending on you to make the same caps.

Also, I didn't notice this earlier, because if you install it in pre-commit with 3.9, then upgrade to 3.10, it will still keep working until the cache is cleared, since it's valid on 3.10, and this only breaks installing, not using. 🤦